lib/get-html-headers.js

```diff
@@ -35,8 +35,18 @@ function rankify(headers, max) {
     })
 }
 
+// Remove code blocks so that false headers are not parsed
+function stripMDCodeBlocks(content) {
+  return content
+    .replace(/```([\s\S]*?)```/g, '') // must be before single backtick check
+    .replace(/`([\s\S]*?)`/g, '')
+    .replace(/<code>([\s\S]*?)<\/code>/g, '')
+    .replace(/<pre>([\s\S]*?)<\/pre>/g, '');
```

Comment:
What's the difference between `[\s\S]` and just `.`?

I'd have thought that "one of either whitespace or non-whitespace" is just "any character".